ganttproject/src/net/sourceforge/ganttproject/search/SearchUiImpl.java

```diff
@@ -110,7 +110,12 @@ public void accept(final List<SearchResult<?>> results) {
       if (results.isEmpty()) {
         return;
       }
-      list.setListData(results.toArray(new SearchResult[0]));
+      SearchResult[] searchResults = results.toArray(new SearchResult[0]);
+      list.setListData(searchResults);
+      int searchResultLength = searchResults.length;
+      if (searchResultLength < 9) {
+        list.setVisibleRowCount(searchResultLength);
```

Comment:
Please add else block which would set visible row count to 10 if we have enough results. Otherwise the dropdown shrinks to very small height after the first search with few results.